src/shared/rpc/methods/open-wallet.ts

```diff
@@ -0,0 +1,14 @@
+import { DefineRpcMethod, RpcRequest, RpcResponse } from '@btckit/types';
+import * as yup from 'yup';
+
+const rpcOpenWalletParamsSchema = yup.object().shape({
+  url: yup.string(),
```

Comment:
should this be optional? Why is it required?

Reply:
Thanks! That was sloppy leftover code from me. Initially I was investigating allowing them to pass a URL path but instead hardcoded it to `RouteUrls.Home`